openedx/core/lib/course_tabs.py

```diff
@@ -0,0 +1,47 @@
+"""
+Tabs for courseware.
+"""
+from openedx.core.lib.api.plugins import PluginManager
+
+_ = lambda text: text
+
+
+# Stevedore extension point namespaces
+COURSE_TAB_NAMESPACE = 'openedx.course_tab'
+
+
+class CourseTabPluginManager(PluginManager):
+    """
+    Manager for all of the course tabs that have been made available.
+
+    All course tabs should implement `CourseTab`.
+    """
+    NAMESPACE = COURSE_TAB_NAMESPACE
+
+    @classmethod
+    def get_tab_types(cls):
+        """
+        Returns the list of available course tabs in their canonical order.
+        """
+        def compare_tabs(first_type, second_type):
+            """Compares two course tabs, for use in sorting."""
+            first_priority = first_type.priority
+            second_priority = second_type.priority
+            if not first_priority == second_priority:
+                if not first_priority:
```

Comment:
Priorities are represented as integers, which currently are all > 0 or `None`.

I do think we should be more explicit about `is None` vs. `falsy`, so I'll clean that up.